azure-pipelines.yml

```diff
@@ -20,10 +20,6 @@ trigger:
       - config/*
       - .github/*
 
-  tags:
```

Comment:
We've stopped doing builds on tags when changed the release policy to all stable.
Some repos are still doing it, but this is being removed as the yamls are touched.